sdk/src/client/node_manager/mod.rs

```diff
@@ -58,6 +63,73 @@ impl std::fmt::Debug for NodeManager {
     }
 }
 
+impl ClientInner {
+    pub(crate) async fn get_request<T: DeserializeOwned + Debug + Serialize>(
+        &self,
+        path: &str,
+        query: Option<&str>,
+        need_quorum: bool,
+        prefer_permanode: bool,
+    ) -> Result<T> {
+        #[cfg(not(target_family = "wasm"))]
```

Comment:
for all of these, we could make it codewise non-duplicative spacing the request and rate limit behind the #not(wasm).
Better for maintenance id say

Reply:
done